api/controllers/UserController.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
 
     const attrs = { name, email: email ? email.toLowerCase() : null, email_validated: false, active: false, group }
     if (isModeratorVal) {
-      attrs.role = GroupMembership.Role.MODERATOR // This is ultimately fed to Group.addMembers, which handles mod -> manager
+      attrs.role = GroupMembership.Role.MODERATOR // This is ultimately fed to Group.addMembers, which handles mod -> Coordinator. TODO: fix this
```

Comment:
I think I changed the addMembers function to take inputs like this and give the person all the four responsibilities...

Reply:
indeed, but still want to clean this up